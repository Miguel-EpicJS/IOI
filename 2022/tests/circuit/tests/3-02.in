385045b28fbc1903eca997d513040235e0229e01
807 483 5
-1 0 1 1 2 0 4 5 2 7 7 10 8 3 11 5 15 12 14 10 14 20 11 12 18 16 23 9 4 19 27 19 22 32 23 28 29 16 20 34 24 37 36 30 8 41 30 15 47 48 33 32 24 46 13 33 49 35 45 43 31 31 56 59 17 22 21 61 62 62 67 37 65 46 55 43 67 60 49 59 48 71 25 61 44 17 72 70 85 84 41 50 64 65 3 44 87 45 88 76 75 34 82 81 91 83 96 68 80 68 54 82 86 94 25 96 39 104 78 28 103 94 80 93 47 115 115 69 90 78 95 85 110 100 75 108 51 127 95 87 118 69 124 81 122 88 114 64 116 105 120 118 111 83 132 102 131 106 27 141 107 124 161 130 121 147 135 130 113 150 84 109 123 138 101 171 147 36 145 153 106 154 181 102 151 144 142 164 90 152 140 160 177 186 21 143 140 56 166 198 108 151 160 136 110 168 60 184 187 127 156 156 188 174 204 137 199 193 70 155 77 176 205 194 143 180 131 38 142 129 183 196 158 229 228 231 226 189 125 211 217 132 159 155 178 212 200 229 113 235 157 166 193 226 190 242 204 233 165 247 220 179 232 192 243 220 191 242 262 201 121 128 210 251 227 145 271 197 239 161 216 227 225 228 162 135 162 149 198 282 269 243 206 258 279 210 262 296 290 186 284 167 251 184 252 252 278 223 301 297 241 258 259 109 302 285 288 111 296 129 301 308 187 236 281 276 317 114 299 244 318 217 326 304 299 310 207 219 255 269 304 190 171 234 313 314 334 266 343 234 272 290 214 153 72 344 112 348 302 335 297 341 342 307 313 273 334 275 345 265 211 248 320 346 306 265 310 333 282 284 292 311 294 241 344 352 373 207 311 362 281 383 202 180 165 240 355 341 206 98 232 164 257 275 395 173 405 331 394 349 353 305 398 375 285 300 314 400 395 362 386 264 253 315 188 327 393 212 387 293 197 293 404 398 359 380 410 409 419 413 428 407 233 425 349 325 444 411 397 277 421 443 405 350 424 294 433 447 442 381 435 442 446 417 205 351 389 370 391 418 431 468 119 273 472 358 292 286 475 441 236 458 463 367 441 471 454 434 397 378 327 402 484 382 305 390 391 399 484 424 411 478 354 492 317 415 488 480 419 443 502 433 422 477 469 463 316 487 431 460 429 369 451 479 485 365 335 503 414 494 377 495 479 393 308 464 358 526 430 494 430 475 268 392 368 268 364 528 517 481 482 356 541 454 542 549 546 461 531 495 512 524 514 552 461 546 425 492 530 504 473 368 503 141 453 482 490 567 567 477 452 513 569 223 548 337 528 573 504 470 532 556 539 346 560 470 553 538 576 564 558 326 593 267 158 244 559 606 509 574 577 539 586 564 573 337 526 587 520 481 303 328 534 410 596 509 611 625 274 607 617 628 629 598 599 447 589 599 485 614 639 578 633 624 578 639 645 616 594 602 559 558 572 582 640 523 513 570 656 635 563 646 604 646 604 661 590 659 562 633 614 518 662 669 659 597 626 585 547 663 407 678 579 638 673 641 598 638 545 631 688 476 608 681 383 626 664 662 680 692 628 699 402 687 702 686 645 590 458 640 696 620 647 672 696 684 663 701 665 706 670 689 697 717 709 715 695 686 708 710 705 705 635 699 636 723 730 669 725 647 688 673 727 713 676 679 601 736 746 748 732 680 745 742 710 741 726 737 712 727 754 751 649 676 729 739 731 743 761 765 622 763 751 730 764 772 771 775 670 749 777 770 714 773 766 753 719 741 757 785 778 762 749 779 733 783 746 765 792 796 717 799 796 802 788 778 801 6 26 40 38 51 42 58 57 53 66 76 58 74 66 100 93 89 103 92 55 79 97 116 86 73 98 117 99 52 122 120 148 163 154 134 133 168 178 170 159 148 50 157 169 139 112 57 136 214 172 74 123 167 209 134 200 128 177 221 215 237 185 152 203 249 250 238 225 230 202 176 91 235 216 126 133 270 89 254 267 245 280 175 237 213 245 286 277 291 195 272 271 150 257 309 295 260 194 125 261 331 332 221 330 328 324 63 340 146 264 218 355 312 295 196 364 263 192 256 357 333 347 321 369 350 201 332 363 274 340 323 101 376 387 360 363 319 298 361 339 324 381 385 401 222 322 370 379 289 208 399 288 307 375 182 283 409 336 354 388 343 365 420 329 406 338 384 246 417 357 224 423 438 367 439 418 436 401 389 385 459 378 421 240 445 361 466 462 440 471 412 452 366 434 316 287 483 253 422 449 465 489 440 486 366 420 303 467 371 496 493 497 488 502 498 426 450 437 474 459 508 506 512 449 515 516 455 510 499 517 372 462 496 507 524 501 507 529 444 523 525 408 320 427 388 468 403 373 491 519 356 400 540 537 456 408 553 545 416 542 535 547 464 396 532 549 561 555 486 522 557 379 536 568 489 566 469 521 533 435 554 581 543 533 544 490 581 575 516 491 580 473 498 589 572 538 582 579 592 531 592 577 374 457 432 588 534 583 612 607 565 584 603 527 550 595 615 566 601 537 603 556 596 591 551 609 625 505 637 642 511 600 637 587 629 660 448 465 641 613 636 632 651 674 610 649 666 677 648 595 690 612 631 652 627 619 624 644 634 630 694 656 690 671 570 374 643 602 585 576 698 679 571 627 691 616 621 691 707 675 667 685 700 685 655 684 678 711 715 713 692 674 708 668 682 709 724 654 726 697 618 654 718 732 693 723 734 703 731 738 728 665 729 725 605 747 657 744 623 764 758 683 500 761 767 776 760 756 735 650 704 755 653 782 766 658 345 768 759 784 655 757 742 787 716 642 780 653 786 779 711 721 739 737 783 752 789 750 777 791 794 793 791 774 773 790 722 782 720 740 795 800 797 781 769 712 803 801 802 804 806 798 805
0 1 0 1 1 0 0 1 0 0 1 1 0 1 0 1 0 0 1 0 1 0 0 0 0 0 1 0 1 0 1 1 0 0 1 0 0 1 1 0 1 1 1 0 0 0 1 1 1 0 0 1 0 1 1 0 1 0 0 1 0 0 0 1 0 0 0 0 0 0 1 1 0 0 1 1 1 1 1 0 0 0 0 1 1 1 0 0 1 1 1 0 0 0 0 1 1 0 1 1 1 0 0 1 1 0 1 1 0 1 0 1 1 0 0 1 1 1 0 1 0 0 1 1 0 1 0 1 0 1 0 0 1 0 1 0 1 0 1 1 1 1 0 1 0 1 1 1 0 0 1 0 1 0 1 0 1 0 1 0 1 0 0 1 1 1 1 0 1 1 0 1 1 1 0 1 1 1 1 0 1 1 0 1 0 1 1 0 1 1 1 1 1 0 1 1 1 0 1 1 0 1 1 1 0 1 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 1 1 1 1 1 1 0 1 0 1 1 1 1 0 1 0 1 0 1 1 0 1 0 0 1 0 1 1 1 0 0 1 1 1 1 1 0 1 1 1 0 1 1 1 1 1 1 1 1 0 0 1 0 0 1 0 1 0 0 1 1 1 0 0 1 0 1 1 1 0 1 1 0 1 0 0 0 1 1 1 1 1 1 0 1 1 1 1 0 0 1 1 0 0 1 0 0 0 0 1 0 0 1 1 0 0 1 0 0 0 0 1 1 0 1 1 1 0 1 0 1 1 0 0 0 1 0 0 1 0 0 1 1 0 1 1 1 0 0 0 1 0 0 1 0 1 1 0 1 0 1 0 1 0 0 1 0 1 1 1 0 1 1 1 0 0 0 1 1 0 0 1 1 1 1 1 1 0 1 0 1 0 1 0 1 0 1 0 1 1 1 1 1 1 1 0 1 1 0 1 0 0 0 0 1 1 1 1 1 1 0 0 0 1 0 1 1 1 0 0 0 0 0 0 0 0 0 0 1 0 1 0 0 1 0 1 1 1 0 0 1 0 0 0 1 0 0 0 0 1 0 0 0 1 1 1 1 0 0 1 0 1
863 1065
1194 1196
1162 1215
810 1107
918 1079
