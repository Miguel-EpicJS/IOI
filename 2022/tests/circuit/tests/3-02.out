4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
747493058
75435678
304852326
580470868
792077662
